src/clj/harja/kyselyt/paallystys.sql

```diff
@@ -226,9 +239,11 @@ SELECT
     pot2a.kokonaismassamaara,
     pot2a.sideaine,
     pot2a.sideainepitoisuus,
-    pot2a.sideaine2
+    pot2a.sideaine2,
+    pot.luotu as "alkaen"
   FROM pot2_alusta pot2a
- WHERE pot2_id = :pot2_id AND poistettu IS FALSE;
+  JOIN paallystysilmoitus pot ON pot.id = pot2a.pot2_id
```

Comment:
tähän JOINiin on hyvä lisätä AND pot.poistettu IS FALSE, vaikka POT:eja ei periaatteessa pystykään tällä hetkellä mitenkään poistamaan

Reply:
Lisätty